app/src/pages/inside/stepPage/modals/postIssueModal/postIssueModal.jsx

```diff
@@ -123,6 +123,17 @@ const messages = defineMessages({
     defaultMessage: 'Failed to post issue',
   },
 });
+const getAnalyticsData = ({ includeData, includeComments, includeLogs }) => {
```

Comment:
I suggest to move it to the file with analytics events to keep the events and their formatting together.